site/app/templates/GitAuthTokens.twig

```diff
@@ -0,0 +1,63 @@
+{% if new_token is defined and new_token_val is defined %}
+    <div class="content" id="new-token-banner">
+        <p>
+            Your token was created successfully. You will only be able to view
+            it's value once so make sure to save it or you will need to create
+            another one later.
+        </p>
+        <p>
+            NOTE: Do not share this token with others. Treat it like a password.
+        </p>
+        <br>
+        <p><b>Name</b>: {{ new_token.getName() }}</p>
+        {% if new_token.getExpiration() is null %}
+            <p><b>Expiration</b>: Never expires</p>
+        {% else %}
+            <p><b>Expiration</b>: {{ new_token.getExpiration()|date('m/d/Y @ h:i A T') }}</p>
+        {% endif %}
+        <p>
+            <b>Value</b>: {{ new_token_val }}
+            <i class="fas fa-copy" id="token-copy-button" title="Copy to clipboard" data-val="{{ new_token_val }}"></i>
+        </p>
+    </div>
+{% endif %}
+
+<div class="content">
+    <h1>Git Auth Tokens</h1>
+    <button class="btn btn-primary" id="new-auth-token-button">Add Git Auth Token</button>
+    <br><br>
+    {% if tokens|length != 0 %}
+    <table class="table mobile-table">
+        <tr>
+            <th>Name</th>
+            <th>Expiration</th>
+            <th>Action</th>
+        </tr>
+        {% for token in tokens %}
+            <tr>
+                <td>{{ token.getName() }}</td>
+                {% if token.getExpiration() is null %}
+                    <td>Never expires</td>
+                {% else %}
+                    {% if token.isExpired(current_time) %}
+                        <td>EXPIRED on {{ token.getExpiration()|date('m/d/Y @ h:i A T') }}</td>
+                    {% else %}
+                        <td>{{ token.getExpiration()|date('m/d/Y @ h:i A T') }}</td>
+                    {% endif %}
+                {% endif %}
+                <td>
+                    <form action="/git_auth_tokens/revoke" method="POST">
+                        <input type="text" hidden name="csrf_token" value="{{ csrf_token }}">
```

Comment:
`input type="hidden"` is the value preferred for form inputs that won't be shown to users to prevent browsers from doing weird autofill behavior and screenreaders from trying to speak those inputs